ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -247,7 +247,7 @@ mkNameCompItem doc thingParent origName origMod thingType isInfix docs !imp = CI
       where
         argTypes = getArgs typ
         argText :: T.Text
-        argText =  mconcat $ List.intersperse " " $ zipWithFrom snippet 1 argTypes
+        argText = mconcat $ List.intersperse " " $ zipWithFrom (\i t -> T.pack "(_ :: " <> snippet i t <> T.pack ")") 1 argTypes
         snippet :: Int -> Type -> T.Text
         snippet i t = "${" <> T.pack (show i) <> ":" <> showGhc t <> "}"
```

Comment:
I think that the `(_ :: ..)` text should be part of the snippet and not part of the completion, since snippets are an optional feature and can be disabled in config.

Reply:
Yes, my first intention in #1700  is to include brackets into snippets, not outside.

Reply:
Thanks for the feedback,

Is the latest commit more what you had in mind? 

Note:
Including the brackets inside the code snippet means that the brackets are also included in the preselected code after the completion. I think that this is desirable since they might sometimes be redundant.

Reply:
Thanks for the feedback,

Is the latest commit more like what you had in mind? 

Note:
Including the brackets inside the code snippet means that the brackets are also included in the preselected code after the completion. I think that this is desirable since they might sometimes be redundant.

Reply:
Yes, that makes sense